lib/kernel/test/interactive_shell_SUITE.erl

```diff
@@ -1348,12 +1421,12 @@ test_exit_initial(new) ->
       [{putline, ""},
        {expect, "1> $"},
        {putline, "2."},
-       {expect, "2"},
-       {putline,"exit()."},
-       {expect, "Eshell"},
-       {expect, "1> $"},
+        {expect, "2"},
```

Comment:
The indentation in this file is still incorrect.

Reply:
Reverted indentation to what it was before my changes